Samsung/A34/res/xml/power_profile.xml

```diff
@@ -0,0 +1,139 @@
+<?xml version="1.0" encoding="utf-8"?>
+<device name="Android">
+    <item name="ambient.on">110.69</item>
+    <item name="screen.on">91.6</item>
+    <item name="screen.full">230.56</item> public void onItem(Locator locator, String name, float value) throws SAXParseException {<item name="bluetooth.active">22.75</item>
```

Comment:
copy-paste fail here.

ambient.on consuming more than screen.on sounds extremely unlikely to me